hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/OverwriteWithLatestSparkRecordMerger.java

```diff
@@ -42,10 +42,4 @@ public String getMergingStrategy() {
   public Option<Pair<HoodieRecord, Schema>> merge(HoodieRecord older, Schema oldSchema, HoodieRecord newer, Schema newSchema, TypedProperties props) throws IOException {
     return Option.of(Pair.of(newer, newSchema));
   }
-
-  @Override
-  public HoodieRecord.HoodieRecordType getRecordType() {
-    return null;
```

Comment:
Do you know the reason of this being null before?

Reply:
nope